packages/multichain-api-middleware/CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0]
+
 ### Changed
 
+- feat: Add more chain-agnostic-permission utility functions from sip-26 usage ([#5609](https://github.com/MetaMask/core/pull/5609))
```

Comment:
Nit: what dot you thing about dropping feat: prefix and putting this under Added Section instead